sbi/inference/snpe/snpe_a.py

```diff
@@ -397,10 +397,10 @@ def __init__(
             prior: The prior distribution.
         """
         # Call nn.Module's constructor.
-
-        super().__init__(flow, flow._condition_shape)
+        super().__init__()
```

Comment:
please remove the changes to this file to not cause conflicts with the big refactor for SNPE_A.

Reply:
this likely also fix the tests.